core/group_setup.go

```diff
@@ -270,7 +270,7 @@ func (s *setupManager) createAndSend(keys []*key.Identity) {
 		group.GenesisSeed = s.oldGroup.GetGenesisSeed()
 	}
 	s.l.Debugw("", "setup", "created_group")
-	fmt.Printf("Generated group:\n%s\n", group.String()) //nolint
+	s.l.Debugw("Generated group", "group", group.String())
```

Comment:
I think that the overall logging situation can be improved. This came from the desire to take the logs and write them into a file. However, it's currently not possible since fmt.Print-style statements are used in a few places of the code. I can revert this if you think it's better handled at a different time.